security/proto/nodeagent_service.proto

```diff
@@ -13,43 +13,20 @@
 // limitations under the License.
 
 syntax = "proto3";
+import "google/rpc/status.proto";
 
 package istio.v1.auth;
 
+option go_package="istio_v1_auth";
 
-service NodeAgent {
+service NodeAgentService {
 	rpc Check(CheckRequest) returns (CheckResponse);
 }
 
 message CheckRequest {
-        string name = 1;
+	string name = 1;
 }
 
 message CheckResponse {
```

Comment:
Why do you remove this, can you add the reason to the description? Thanks.

Reply:
Done. Thanks.